integreat_cms/cms/models/poi_categories/poi_category.py

```diff
@@ -43,6 +43,50 @@ def name(self):
             else str(_("POI category"))
         )
 
+    # TODO The following methods are copied from abstract_content_model.py. Find a better solution to avoid duplication
+    def get_prefetched_translations_by_language_slug(
+        self, attr="prefetched_translations", **filters
+    ):
+        r"""
+        This method returns a mapping from language slugs to their latest translations of this object
+
+        :param attr: Which attribute should be tried to get the prefetched translations [optional, defaults to ``"prefetched_translations"``]
+        :type attr: str
+
+        :param \**filters: Additional filters to be applied on the translations (e.g. by status)
+        :type \**filters: dict
+
+        :return: The prefetched translations by language slug
+        :rtype: dict
+        """
+        try:
+            # Try to get the prefetched translations (which are already distinct per language)
+            prefetched_translations = getattr(self, attr)
+        except AttributeError:
+            # If the translations were not prefetched, query it from the database
+            prefetched_translations = (
+                self.translations.filter(**filters)
+                .select_related("language")
+                .order_by("language__id")
+                .distinct("language__id")
+                .all()
+            )
+        return {
+            translation.language.slug: translation
+            for translation in prefetched_translations
+        }
+
```

Comment:
```suggestion
```
Fortunately, the POI categories are very simple, so we don't need this complicated caching mechanism (for the other content methods, this is only required because the default prefetching mechanism of Django does not allow to filter the prefetched queryset, so we need to build one prefetched attribute for each filtered queryset combination we want to have for each status etc...)

Reply:
Thanks for the explanation :heart: